README.md

```diff
@@ -1,5 +1,29 @@
-# Momento client-types-php
+# client-sdk-php
 
-:warning: Pre-Alpha :warning:
+:warning: Experimental SDK :warning:
+
+PHP SDK for Momento is experimental and under active development. There could be non-backward compatible changes or
+removal in the future. Please be aware that you may need to update your source code with the current version of the SDK
+when its version gets upgraded.
+---
+
+<br />
+PHP SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by
+traditional caching solutions.
+
+<br/>
+
+# Getting Started :running:
+
+## Requirements
+
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- At least PHP 7
```

Comment:
I'm a php supernoob - is this going to be an acceptable minimum version?  I have no idea how old or new that is :)

Reply:
gRPC requires 7+ so, yes :-)

Reply:
okay but how recent is that?  Is it new enough for use in wordpress etc.?  Just want to understand what things will and won't be open to us